src/app/components/media-viewer-wrapper/media-viewer-wrapper.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, OnInit, ViewEncapsulation } from '@angular/core';
 import { WindowService } from '@hmcts/ccd-case-ui-toolkit';
 import { Observable } from 'rxjs';
 import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
+import { SessionStorageService } from 'src/app/services';
```

Comment:
Shouldn't import from `src/`.

Reply:
Changed to relative path. Still waiting for Phil's path change to go in...